Features/Shop.feature

```diff
@@ -1,8 +1,118 @@
-﻿Feature: Shop Page
+﻿@PageSetup
+
+Feature: Shop
+
+As a User, I want to group products into tabs,
+I can filter products by values on those tabs,
+I can add items to the cart, 
+and remove added items from the cart.
+
+Scenario: Verify every Tab <h1>:
+
+	#Arrange:
+	Given I am on 'shop' page
+	#Act:
+	When I click on '<Tab Name>' tab
+	#Assert:
+	Then I see '<h1>' heading is displayed
+
+Examples:
+	| Tab Name           | h1                |
+	| Сонячні панелі     | Сонячні панелі    |
+	| Інвертори          | Сонячні інвертори |
+	| Акумулятори        | Акумулятори       |
+	| Контролери заряду  | Контролери заряду |
+	| Системи кріплення  | Системи кріплення |
+	| Кабель і комутація | Сонячний кабель   |
+
+
+#Verify that catalog filter works correctly
+
+Scenario: Verify Filter on the tab:
+
+	#Arrange:
+	Given I am on 'shop/<Tab>' page
+	#Act:
+	When I click on Filter button
+	And I click on '<FilterValue>' checkbox
+	#Assert:
+	Then I see '<FilterValue>' filtered products
```

Comment:
the fact that you see filtered product does not mean that filtering works. What if this product was on the page before filtering and filtering itself is broken?